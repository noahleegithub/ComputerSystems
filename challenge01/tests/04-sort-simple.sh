mkdir -p tmp
sort -o tmp/sorted.txt tests/sample.txt
cat tmp/sorted.txt
