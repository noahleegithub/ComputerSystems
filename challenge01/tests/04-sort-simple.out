ambassador
flown
iguana
introduces
plenty
pontiff
reactivates
remorselessly
revolutionized
underplays
